docker/entrypoint.sh

```diff
@@ -0,0 +1,47 @@
+#! /bin/bash
+clear
+
+# check for + link mounted libraries:
+if [ -d /mnt/sendgrid-python ]
+then
+	rm /root/sendgrid
+	ln -s /mnt/sendgrid-python/sendgrid
+	echo "Linked mounted sendgrid-python's code to /root/sendgrid"
+fi
+if [ -d /mnt/python_http_client ]
+then
+	rm /root/python_http_client
+	ln -s /mnt/python-http-client/python_http_client
+	echo "Linked mounted python-http-client's code to /root/python_http_client"
+fi
+
+SENDGRID_PYTHON_VERSION=$(python2.7 -c 'import sendgrid; print(sendgrid.__version__)')
+echo "Welcome to the sendgrid-python docker (version $SENDGRID_PYTHON_VERSION)"
+echo
+
+if [ "$1" != "--no-mock" ]
+then
+	echo "Starting Prism in mock mode. Disable this by running this container with --no-mock."
```

Comment:
Ah, I see you reference it as passthrough mode later. We should be consistent and say mock/no-mock or something similar.